DATABASE/stored_procedures/orders/cancelOrder.sql

```diff
@@ -0,0 +1,9 @@
+CREATE OR ALTER PROCEDURE cancelOrderById(
+@id VARCHAR(200)
+)
+AS
+BEGIN
+SELECT * FROM orders WHERE id=@id
+UPDATE orders SET orderStatus= 'canceled'
```

Comment:
This will update all orders to canceled.
Remove select, keep update but with where clause